api/v1alpha1/watermarkpodautoscaler_types.go

```diff
@@ -93,6 +93,9 @@ type WatermarkPodAutoscalerSpec struct {
 	Metrics []MetricSpec `json:"metrics,omitempty"`
 	// +kubebuilder:validation:Minimum=1
 	MinReplicas *int32 `json:"minReplicas,omitempty"`
+	// MinAvailableReplicaPerc indicates the minimum percentage of replicas that need to be available in order for the
+	// controller to autoscale the target.
+	MinAvailableReplicaPerc int32 `json:"minAvailableReplicaPerc,omitempty"`
```

Comment:
should we put the full name 
```suggestion
	MinAvailableReplicaPercentage int32 `json:"minAvailableReplicaPercentage,omitempty"`
```

Reply:
also maybe add kubebuilder comment constraint to limit the allow value